doc/02-Installation.md

```diff
@@ -0,0 +1,18 @@
+# <a id="Installation"></a>Installation
+
+## Requirements
+
+* Icinga Web 2 (&gt;= 2.8 RC1)
+* PHP (&gt;= 5.6, preferably 7.x)
```

Comment:
IIRC a new dependency is php-redis, or is that provided somewhere else?

For RHEL8, it seems that php-pecl-redis is not yet packaged.
https://serverfault.com/questions/997028/where-is-the-redis-module-for-php-on-centos-8-php-pecl-redis